src/java.base/share/classes/java/lang/Byte.java

```diff
@@ -90,7 +90,7 @@
      * @see java.lang.Integer#toString(int)
      */
     public static String toString(byte b) {
-        return Integer.toString((int)b, 10);
+        return Integer.toString(b);
```

Comment:
This change makes sense given the evidence that the radix tests in `Integer.toString(int, int)` are not elided. Same for the equivalent change in `Short.java`.